submissions/Border Walls/border-walls.md

```diff
@@ -0,0 +1,11 @@
+
+Building a border wall farther out than the edge of your factory has numerous advantages. First and foremost, it allows you to choose where you want to deal with bugs. If you clear out to a choke point and then hold that choke, you don't have to worry about attacks from that direction - and if you do this in all directions, you don't have to worry about attacks at all. Pushing out past your pollution boundary means that bugs won't even attack at these defenses: the only thing the turrets are there for is to stop the expansion of bugs into already-cleared territory. By using natural barriers such as cliffs and water to aid your defenses, you can cut off swathes of the map with very little use of resources: I would estimate somewhere between forty and eighty percent of my border wall tends to be naturally-formed, impassible obstacles. With trees and tiles absorbing my pollution instead of biter spawners, I don't even get very frequently attacked - and this is crucial, because it allows me to throw down a couple of hundred rounds of piercing ammo at a chokepoint and leave it alone for several hours.
+
+The second major advantage of building a border wall this far out is that it doesn't inhibit your factory. When you need to expand to an iron patch, it's likely you've already got one within your borders. When you need space for a massive oil-refining depot, you've got space to do that, because you've got three or four times the radius of your factory clear in all directions. Even when you start setting up rails to distant outposts, it's likely those are - and should be - within your borders, so you don't have to defend them any more than the border wall already does. Rather than relocating a wall each time you expand your factory, you relocate your wall to claim additional territory, and then expand your factory as you see fit without worrying about attacks.
+
+The science of border walls has changed my life. I used to defend outposts with rows upon rows of laser turrets, struggling to keep them repaired and supplied as bugs did incremental damage. Now, I simply push out to natural chokes and keep them manually supplied with ammo: using only a couple dozen rounds per hour, such natural chokes survive far longer out past my pollution boundary than they ever could within it. In the late game, when I finally have access to artillery, I even put artillery turrets along my border walls - and this allows me to expand without ever having to clear bugs manually. The slowly creeping wall reduces the amount of damage that I have to deal with down to almost zero - and at the same time, I get to keep bugs toggled on, allowing me to watch as they ever-so-occasionally break against my carefully made defenses.
```

Comment:
Maybe finish the submission with a gif of biters attacking a border wall setup? Might help illustrate how much of a difference a choke point makes.